airbyte-cdk/python/airbyte_cdk/sources/declarative/stream_slicers/datetime_stream_slicer.py

```diff
@@ -170,14 +171,11 @@ def _partition_daterange(self, start, end, step: datetime.timedelta):
         return dates
 
     def _get_date(self, cursor_value, default_date: datetime.datetime, comparator) -> datetime.datetime:
-        cursor_date = self.parse_date(cursor_value or default_date)
```

Comment:
no need to call parse_date because the cursor_value is already a datetime object